examples/secrets-management-with-csi-secrets-driver-aws/README.md

```diff
@@ -0,0 +1,131 @@
+# EKS Cluster Deployment with new VPC
```

Comment:
Instead of creating a whole new example just for these two addons, instead can we just add their implementation into the https://github.com/aws-ia/terraform-aws-eks-blueprints/tree/main/examples/complete-kubernetes-addons

Reply:
@askulkarni2 / @kcoleman731 what are your thoughts on this as we continue to expand the number of addons?

Reply:
I agree with @bryantbiggs . We should start moving the individual Add-on examples to `[complete-kubernetes-addons](https://github.com/aws-ia/terraform-aws-eks-blueprints/tree/main/examples/complete-kubernetes-addons)` . This will help us to manage limited exmaples

Reply:
We can add some more details to `docs` `.md` files as well.  New users should be able to goto docs to findout the usage of this Add-on.